app/models/hyrax/file_metadata.rb

```diff
@@ -76,19 +76,19 @@ def self.for(file:)
       new(label: file.original_filename,
           original_filename: file.original_filename,
           mime_type: file.content_type,
-          use: file.try(:use) || [::Valkyrie::Vocab::PCDMUse.OriginalFile])
+          type: file.try(:type) || [Hyrax::FileSet.original_file_use])
```

Comment:
could we use a Dry::Struct/Types default here? something like `attribute :type, ::Valkyrie::Types::Set, default: [Hyrax::FileSet.original_file_use]`?

Reply:
@no-reply This is the only requested change I didn't make.  I don't have time to work on this today, so I'd like to punt this requested change to later.  Open issue #4226 to track this request.